ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/AuthEndpoint.java

```diff
@@ -96,26 +98,30 @@ public Response auth(
       @QueryParam("response_type") String responseType,
       @QueryParam("client_id") String clientId,
       @QueryParam("redirect_uri") String redirectUri,
-      @QueryParam("nonce") String nonce) {
+      @QueryParam("nonce") String nonce,
+      @HeaderParam("Accept-Language") @DefaultValue("en-GB") String acceptLanguage) {
```

Comment:
to be precise, our default is US English -> https://apiguidelines.oviva.io/#103

Reply:
Yes I know that our default is US English, but we used en-GB for con the `content-engine` in the past. That's why I used the latter. Can change it to US though